Oscil.h

```diff
@@ -157,9 +157,9 @@ class Oscil
 	each direction. This fixed point math number is interpreted as a SFix<15,16> internally.
```

Comment:
Optimization question (maybe for later): Aren't those 15 NI bits fully ineffective, 16 fractional bits alone would fully suffice? (Sign might be a problem, I imagine)

Reply:
Yes, I thought of that too. Some examples (for instance FMSynth) do wander further than pure fractional, but as it rolls back, only fractional should be enough (and further is taking care of  smoothly by the overflow). Indeed the sign is a problem, but maybe an extra function taking SFix<0,15>, so sacrificing one bit of precision, could be interesting. Might not be straightforward to make the compiler choose between the two though. 

I would suggest taking that a bit further down the pipeline as this is more optimisation of legacy code that directly related to FixMath.

Reply:
Agreed. Maybe collection things like this somewhere would help keeping track, though. Perhaps in an issue, or by marking them as "TODO: optimize" in the code.